src/components/button-toggle/button-toggle.stories.js

```diff
@@ -33,7 +33,7 @@ function makeStory(name, themeSelector) {
     };
 
     return (
-      [
+      <>
```

Comment:
Using React Fragments like this Ie. declaring what look like empty elements causes the Storybook addon to output this message.
```
"Unknown" Component
No propTypes defined!
```
It's better to use a `<div>` here, as there's no problem for us to add the extra element to the DOM and it cleans up that Unknown Component output.

Reply:
@damianOfKyan this issue has reappeared. Please could you resolve it.

Reply:
fixed in d1a866b